packages/plugin-dom-helpers/src/DomHelpers.ts

```diff
@@ -175,6 +175,19 @@ export class DomHelpers<T extends JWPluginConfig = JWPluginConfig> extends JWPlu
         }
         await this.editor.dispatcher.dispatchHooks('@redraw');
     }
+    /**
+     * Prepend a DOM Node inside the child of another.
```

Comment:
"Prepend a DOM Node to another DOM Node's children." - otherwise it seems like you want to prepend it to that other node's child.